config/crd/standard/gateway.networking.k8s.io_grpcroutes.yaml

```diff
@@ -152,7 +152,7 @@ spec:
                 maxItems: 16
                 type: array
               parentRefs:
-                description: |+
+                description: |-
```

Comment:
@robscott I'm wondering whether we should consider adding a single linebreak at the end of a description.

Reply:
I reviewed other CRDs, and I noticed that it's not typical for the description to end with a `\n`.